Makefile

```diff
@@ -8,7 +8,7 @@ GOFMT = gofmt
 VERSION = dev-$(shell date +%FT%T%z)
 
 BINARY = kubeapps
-GO_PACKAGES = $(IMPORT_PATH)/cmd $(IMPORT_PATH)/pkg/...
+GO_PACKAGES = $(IMPORT_PATH)/cmd/kubeapps $(IMPORT_PATH)/pkg/...
```

Comment:
it should be `$(IMPORT_PATH)/cmd/...` That way we can add more commands in cmd/ folder